Integration/GoToAbstractIntegration.php

```diff
@@ -133,10 +145,86 @@ public function getOrganizerKey(): string
         return $keys['organizer_key'];
     }
 
-    public function getAccountKey(): string
+    public function getAccountKey(): ?string
     {
         $keys = $this->getKeys();
 
-        return $keys['account_key'];
+        return $keys['account_key'] ?? null;
+    }
+
+    public function fetchAccountKey(string $accessToken): string
```

Comment:
it look like will work very well and I get your idea, but I think that we can optimize code.

But we are repeating big part of code, my ideal is create one method fetchAll(string $accessToken)

and inside you use try catch, because it is in this point that your code can broke without your control...

somthing like that:

```
public function fetchKeys(string $accessToken): string
{
try {
$accountData = $this->fetchGoToData($accessToken, $this->getAccountUrl());
}catch (\Exception $e) {
throw new \Exception('Missing correct data');
}
if (!isset($organizerData[0]['organizerKey']) || !isset($accountData['accountKey'])) {
           return false;
}
....
return [
'account_key' => (string) $accountData['accountKey'],
'organizer_key' => $accountData['accountKey']
]
}

public function blaba()
{
$parsed  = $this->fetchKeys($parsed['access_token']);
if(!parsed) {
log...
return false,
}
....
```